lib/mjml.rb

```diff
@@ -27,14 +27,25 @@ def self.discover_mjml_bin
     return mjml_bin if check_version(mjml_bin)
 
     # Check for a local install of MJML binary
-    installer_path = (`npm bin` || `yarn bin`).chomp
+    installer_path = bin_path_from('npm') || bin_path_from('yarn')
+    unless installer_path
+      puts Mjml.mjml_binary_error_string
+      return nil
```

Comment:
I am not super happy that I am repeating the error output here.

In general I think one could invest some more work to clean up the finder code. What I also noticed while debugging the failing CI is that "mjml was not found" is indistinguishable from "mjml was found, but it is an unsupported version".

Waiting for your feedback, I would think it is "good enough" now (as in: both yarn and npm alone are enough to use this gem).

(FYI: I'll probably not be able to respond to comments in the next two weeks, but I will respond eventually :) )